app/gui/src/presenter/project.rs

```diff
@@ -243,8 +302,27 @@ impl Project {
         let view = &model.view.frp;
         let breadcrumbs = &model.view.graph().model.breadcrumbs;
         let graph_view = &model.view.graph().frp;
+        let project_list = &model.view.project_list();
 
         frp::extend! { network
+            project_list_ready <- source_();
+
+            project_list.grid.reset_entries <+ project_list_ready.map(f_!([model]{
+                let cols = 1;
+                let rows = model.available_projects.borrow().len();
+                (rows, cols)
+            }));
+            entry_model <- project_list.grid.model_for_entry_needed.map(f!([model]((row, _)) {
+                model.available_projects.borrow().get(*row).map(|(name, _)| (*row, 0, name.clone_ref()))
```

Comment:
this is super complex. Also, the `0` is magic

Reply:
Fixed